.github/workflows/push_snapshots.yml

```diff
@@ -2,13 +2,42 @@ name: Push snapshot to Maven repo
 
 on:
   workflow_dispatch:
-#  push:
-#    branches:
-#      - main
+  push:
+    branches:
+      - main
 
 jobs:
+  build_macos_rust_libs:
+    name: Build native library for MacOS
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install stable
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - name: Set up Java
+        uses: actions/setup-java@v2
+        with:
+          java-version: 8
+          distribution: 'adopt'
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@e6e38bacfdf1a337459f332974bb2327a31aaf4b
```

Comment:
Any specific reason to use the git hash rather than their released version?

Reply:
No reason actually. I changed to tags.